parsons/databases/redshift/rs_table_utilities.py

```diff
@@ -428,7 +428,8 @@ def get_columns_list(self, schema, table_name):
         `Returns:`
             A list of column names.
         """
-
+        schema = f'"{schema}"'
```

Comment:
What if someone was already working around this issue by passing in the name of the schema wrapped in quotes? Should we check if `schema` and `table_name` are already wrapped in quotes, and only add them if they aren't already?

Reply:
Ah, good catch. Yes, I think you are right.